frontend/src/metabase/public/containers/PublicApp/PublicApp.unit.spec.tsx

```diff
@@ -101,4 +111,25 @@ describe("PublicApp", () => {
     expect(screen.queryByText(/Powered by/i)).not.toBeInTheDocument();
     expect(screen.queryByText(/Metabase/)).not.toBeInTheDocument();
   });
+
+  describe("theming", () => {
+    it("renders correctly without a theme parameter", () => {
+      setup();
+
+      const embedFrame = screen.getByTestId("embed-frame");
+
+      // eslint-disable-next-line jest-dom/prefer-to-have-class
+      expect(embedFrame.className).not.toEqual(
+        expect.stringContaining("Theme--"),
+      );
```

Comment:
Unfortunately couldn't make this work with `toHaveClass` or `not.toHaveClass` :/